File: KotlinProject/src/week2/Achievements.py
people = {
    "Boole": "Boolean Logic",
    "De Morgan": "De Morgan's Law",
    "Shannon": "Theorize Logic",
    "Atanasoff": "Theorize Digital Comp",
    "Mauchly": "UNIVAC",
    "Von Neumann": "Computer Architecture"
}

while True:
    name = input('Enter a name (or q to quit): ')
    if name.lower() == 'q':
        break
    if name in people:
        print(people[name])
    else:
        print('Name not found')

File: KotlinProject/src/week2/Achievements.java
package week2;

import java.util.Map;
import java.util.Scanner;
import java.util.TreeMap;

public class Achievements {

    public static void main(String[] args) {
        ask();
    }

    public static void ask() {
        Map<String, String> dictionary = new TreeMap<>();

        // Adding key-value pairs to the dictionary
        dictionary.put("Boole", "Boolean Logic");
        dictionary.put("De Morgan", "De Morgan's Law");
        dictionary.put("Shannon", "Theorize Logic");
        dictionary.put("Atanasoff", "Theorize Digital Comp");
        dictionary.put("Mauchly", "UNIVAC");
        dictionary.put("Von Neumann", "Computer Architecture");

        Scanner scanner = new Scanner(System.in);

        while(true) {
            System.out.print("Enter an name: ");
            String userInput = scanner.nextLine();
            if (userInput.equals("q")) { break; }
            System.out.println(dictionary.getOrDefault(userInput, "Name not found"));
        }
        scanner.close();
    }
}
